src/main/resources/hudson/os/windows/Messages.properties

```diff
@@ -22,8 +22,8 @@
 
 ManagedWindowsServiceLauncher.DisplayName=\
   Let Jenkins control this Windows slave as a Windows service
-ManagedWindowsServiceLauncher.DotNetRequired={0} [windows-slaves] .NET Framework 2.0 or later is required on this computer to run a Jenkins slave as a Windows service
-ManagedWindowsServiceLauncher.InstallingSlaveService={0} [windows-slaves] Installing the Jenkins slave service
+ManagedWindowsServiceLauncher.DotNetRequired={0} [windows-slaves] .NET Framework 2.0 or later is required on this computer to run a Jenkins agent as a Windows service
+ManagedWindowsServiceLauncher.InstallingSlaveService={0} [windows-slaves] Installing the Jenkins agent service
```

Comment:
Ah, ok. I thought it was maybe a Windows service name or something.

But end users shouldn't ever really care about the artifact ID — the plugin name has been updated, which will be shown on plugins.jenkins.io and the Plugin Manager.  So it seems ok to replace?